android/config.gradle

```
@@ -31,10 +43,17 @@ if (pspdfkitVersion == null || pspdfkitVersion == '') {
 
 ext.pspdfkitMavenModuleName = 'pspdfkit'
 
-ext.pspdfkitFlutterVersion = '1.10.3'
+// Get our library version from the top level pubspec.yaml file.
+def pubspecFilePath = '../pubspec.yaml'
+def pubspecFile = rootProject.file(pubspecFilePath)
+if (!pubspecFile.exists()) {
+    throw new GradleException("'pubspec.yaml' file not found. Expected at the top level of the Flutter repository.")
+}
+def pubspecYaml = new Yaml().load( pubspecFile.newInputStream() )
```

Please don't use spaces for function arguments.
```suggestion
def pubspecYaml = new Yaml().load(pubspecFile.newInputStream())
```

---

Oops, good spot.